### PR TITLE
Feat/add pie and bar charts 73 74

### LINKED ISSUE #73: Create Bar Chart Components as per Design
## Implement Bar Chart Components Based on New Design

Image

**Description:**
Create bar chart components in the UI kit as introduced in Edwin's latest design specifications. These components should be flexible enough to accommodate future design variations.

**Acceptance Criteria:**
- New bar chart components match Edwin's design specifications.
- Components are responsive and adapt to various screen sizes.
- Documentation is updated to include examples and usage guidelines.

### LINKED ISSUE #74: Create Pie Chart Component
## Add Pie Chart Component to UI Kit

Image

**Description:**
Create a pie chart component within our UI kit, ensuring it is versatile for different data representations and integrates seamlessly with our design language.

**Acceptance Criteria:**
- The pie chart component visually aligns with our UI design principles.
- It supports dynamic data input and updates.
- Include interactive elements (e.g., tooltips) for a better user experience.
- Update documentation with implementation details and examples.

### PR DESCRIPTION
**Pull request by Torresmorah**
### What does this PR do?

- Resolve #73
- Resolve #74
- Refactor Dashboard to receive the main content and the items from the right column as params
- Add simple tooltip to the Bar and Pie Chart
- Update to 0.0.46 version

### Steps to test

1. Run the project locally
2. Go to Dashboard component
3. Check the three examples

### Screenshots

**Stories**

imagen

**Desktop breakpoint**

imagen
imagen
imagen
imagen

**Mobile breakpoint**

imagen
imagen

### CODE REVIEW

**Comment by xavier506** [comment]
@Torresmorah  please resolve conflict

File: src/stories/BarChart.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { Story } from "@storybook/react";
+
+import { BarChart, IBarChart, ChartSeriesType } from "../components/BarChart";
+
+export default {
+  title: "Charts/BarChart",
+  component: BarChart,
+};
+
+const formatValue = (value: number, options?: Intl.NumberFormatOptions) => {
```
Comment (leisterfrancisco) [inline src/stories/BarChart.stories.tsx:11]: avoid the usage of number, instead, use bigint.

Please check a function in utils/functions that have a similar method to format a number.

File: src/stories/Dashboard.stories.tsx [hunk 2]
```diff
@@ -67,6 +74,13 @@ const bottomContainerItems = [
   { label: "Total Lent", value: 20000, fluctuation: "+3.4%" },
 ];
 
+const rightContainerItems = [
+  { label: "Active Loans", value: 1200 },
+  { label: "All Repaid Loans", value: 2500 },
+  { label: "Interest Paid", value: 200 },
+  { label: "Loan to Value Ratio", value: "75%" },
```
Comment (leisterfrancisco) [inline src/stories/Dashboard.stories.tsx:81]: i18n

File: src/stories/Dashboard.stories.tsx [hunk 3]
```diff
@@ -76,30 +90,106 @@ const formatValue = (value: number, options?: Intl.NumberFormatOptions) => {
...
+  rightLabel: "Stats",
+  rightContainerItems: rightContainerItems1,
+  formatValue,
+  content: <Charts />,
 };
 
-export const DashboardItem = Template.bind({});
-DashboardItem.args = {};
+const rightContainerItems2 = [
+  { label: "Loans Ready to Borrows", value: 2500 },
```
Comment (leisterfrancisco) [inline src/stories/Dashboard.stories.tsx:183]: i18n

File: src/stories/PieChart.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+const Template: Story<IPieChart> = (args) => {
+  return <PieChart {...args} />;
+};
+
+const pieChartData: PieDataType = [
+  { name: "Active", value: 135 },
+  { name: "Claimed", value: 50 },
+  { name: "Available", value: 65 },
+];
```
Comment (leisterfrancisco) [inline src/stories/PieChart.stories.tsx:27]: i18n

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: src/stories/Dashboard.stories.tsx [hunk 5]
```diff
@@ -76,30 +90,106 @@ const formatValue = (value: number, options?: Intl.NumberFormatOptions) => {
...
+  { label: "Total Borrows Repaid", value: 200 },
```
Comment (xavier506) [inline src/stories/Dashboard.stories.tsx:184]: Total Loans Repaid

File: src/stories/Dashboard.stories.tsx [hunk 6]
```diff
@@ -76,30 +90,106 @@ const formatValue = (value: number, options?: Intl.NumberFormatOptions) => {
...
+  { label: "Total Debt", value: 800 },
```
Comment (xavier506) [inline src/stories/Dashboard.stories.tsx:185]: Total Loans

File: src/stories/Dashboard.stories.tsx [hunk 7]
```diff
@@ -76,30 +90,106 @@ const formatValue = (value: number, options?: Intl.NumberFormatOptions) => {
...
+  { label: "Next Borrow to Reach Maturity", value: 'Pool B' },
```
Comment (xavier506) [inline src/stories/Dashboard.stories.tsx:186]: Next Loan to Reach...

File: src/stories/Dashboard.stories.tsx [hunk 8]
```diff
@@ -76,30 +90,106 @@ const formatValue = (value: number, options?: Intl.NumberFormatOptions) => {
...
+          formatValue={formatValue}
+          series={barChatSeries}
+          colors={barChartColors}
+          data={poolNames}
+        />
+      }
     />
-  );
+    <ChartContainer
+      chartSubtitle="Pool by Status"
```
Comment (xavier506) [inline src/stories/Dashboard.stories.tsx:158]: Pools by Status

**Review by leisterfrancisco — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**